terraform/aws/implementation/modules/eks/main.tf

```diff
@@ -31,19 +37,28 @@ module "eks-cluster" {
         { namespace = "default" }
       ]
       subnet_ids = var.private_subnet_ids
+      iam_role_additional_policies = {
+        logging = aws_iam_policy.cloudwatch_policy.arn
+      }
     }
     karpenter = {
       selectors = [
         { namespace = "karpenter" }
       ]
       subnet_ids = var.private_subnet_ids
+      iam_role_additional_policies = {
+        logging = aws_iam_policy.cloudwatch_policy.arn
+      }
     }
     kube_system = {
       name = "kube-system"
       selectors = [
         { namespace = "kube-system" }
       ]
       subnet_ids = var.private_subnet_ids
+      iam_role_additional_policies = {
+        logging = aws_iam_policy.cloudwatch_policy.arn
```

Comment:
Good to see we are adding logging to help us try to troubleshoot things in the future.